polkadot-cli/src/lib.rs

```diff
@@ -19,18 +19,28 @@
 #![warn(missing_docs)]
 
 extern crate env_logger;
-extern crate polkadot_client as client;
-extern crate substrate_executor as executor;
+extern crate ed25519;
+extern crate triehash;
+extern crate substrate_codec as codec;
+extern crate substrate_state_machine as state_machine;
+extern crate substrate_client as client;
 extern crate substrate_primitives as primitives;
-extern crate polkadot_rpc_servers as rpc;
+extern crate substrate_runtime_io as runtime_io;
+extern crate substrate_rpc_servers as rpc;
+extern crate polkadot_primitives;
+extern crate polkadot_executor;
+extern crate polkadot_runtime;
```

Comment:
most of these crates aren't used in this binary. why pull them in?

Reply:
i think most of them are used - i only managed to pull one unused one out.